single-node-k8s.sh

```diff
@@ -64,6 +64,7 @@ function set_env() {
     export GO111MODULE="on"
     export TEST_COVERAGE="stdout"
     export VM_DRIVER="kvm2"
+    export MEMORY="8192"
```

Comment:
@nixpanic can you also increase the cpu for minikube vm

Reply:
yes, increased by default through #1312